src/material-examples/datepicker-api/datepicker-api-example.html

```diff
@@ -0,0 +1,5 @@
+<md-input-container class="example-full-width">
+  <input mdInput [mdDatepicker]="picker" placeholder="Choose a date">
+</md-input-container>
+<md-datepicker #picker></md-datepicker>
+<button md-raised-button (click)="picker.open()">Open</button>
```

Comment:
a number of files are missing newline at end, you should configure your editor to insert it